examples/websites/README.md

```diff
@@ -0,0 +1,53 @@
+#ipfs for websites
+### A short guide to hosting your site on ipfs
+
+Adding your static website to ipfs is quite simple! Simply turn on your daemon:
+```bash
+$ ipfs daemon
+```
+
+And add the directory containing your website:
+```bash
+$ ls mysite
+index.html
+$ ipfs add -r mysite
+added QmbRyftGAtNjXs7hWZUTLyp27qrpgRUXKEwXCVPBueZGek mysite/index.html
+added QmYAMp9ptyn3Bv8ijoYoYAAQHFCfpZ1NKRNedkEWrR9DPH mysite/
+```
+
+The very last hash next to the folder name is the one you want, lets call it
+`$SITE_HASH` for now.  
+
+Now, you can test it out locally by opening `http://localhost:8080/ipfs/$SITE_HASH`
+in your web browser! Next, to view it coming from another ipfs node, you can try
+`http://gateway.ipfs.io/ipfs/$SITE_HASH`. Cool, right?  But those hashes are
+kinda ugly. Lets look at some ways to get rid of them.
+
+First, you can do a simple DNS TXT record, containing `dnslink=/ipfs/$SITE_HASH`.
+Once that record propogates, you should be able to view your site at
+`http://localhost:8080/ipns/your.domain`. Now thats quite a bit cleaner.
+
+Next, you might be asking "well what if i want to change my website, DNS is slow!"
+Well let me tell you about this little thing called Ipns. Ipns is the Interplanetary
```

Comment:
maybe say: "a little thing called IPNS (note the "n"). IP_N_S is the InterPlanetary Naming System, you ..."